src/warnet/tank.py

```diff
@@ -26,6 +26,8 @@
 
 
 class Tank:
+    DEFAULT_BUILD_ARGS = "--disable-tests --with-incompatible-bdb --without-gui --disable-bench --disable-fuzz-binary --enable-suppress-external-warnings --enable-debug "
```

Comment:
no real opinion on this, but why define this global inside the class instead of further up with the other globals?

Reply:
Not really sure/no strong opinion either. Variables defined here exist once shared between all instances of the class, but only within the class namespace, which just seemed like exactly what we would want?